Services/UserService/Utils/EmailSender.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Net;
+using System.Net.Mail;
+using UserService.Interfaces;
+
+namespace UserService.Utils
+{
+    public static class EmailSender
+    {
+        public static void SendEmail(string email,ISecretTokenEngine secretTokenEngine)
+        {
+            var from = new MailAddress("traidplatform@mail.ru", "Trading Station");
```

Comment:
Please change it to "tradeplatform" or "tplatform" or anything else without grammar mistakes, this one hurts a lot